### PR TITLE
Added 'mafia test' & 'mafia repl'

### PR DESCRIPTION
**Pull request by jacobstanley**
:boom:

### CODE REVIEW

File: main/mafia.hs [hunk 1]
```diff
@@ -69,6 +73,12 @@ commands =
  [ command' "build" "Build this project, including all executables and test suites."
             (MafiaBuild <$> many pCabalArgs)
 
+ , command' "test" "Test this project, by default this runs all test suites."
+            (MafiaTest <$> many pCabalArgs)
+
+ , command' "repl" "Start the repl, by default on the the main library source."
```
Comment (tranma) [inline main/mafia.hs]: "the the"

Comment (jacobstanley) [inline main/mafia.hs]: Cheers!

https://github.com/ambiata/cabal-extras/blob/master/src/cabal#L18

Comment (tranma) [inline main/mafia.hs]: lol :shipit:

File: script/mafia [hunk 2]
```diff
@@ -27,31 +27,37 @@ run_upgrade () {
...
   MAFIA_VERSION=$(awk '/^# Version: / { print $3; exit 0; }' $0)
-  MAFIA_PATH=$HOME/.ambiata/mafia/bin/mafia-$MAFIA_VERSION
 
-  [ -f "$MAFIA_PATH" ] || {
-    # Create a temporary directory which will be deleted when the script
-    # terminates. Unfortunately `mktemp` doesn't behave the same on
-    # Linux and OS/X so we need to try two different approaches.
-    MAFIA_TEMP=$(mktemp -d 2>/dev/null || mktemp -d -t 'exec_mafia')
-    trap "rm -rf \"$MAFIA_TEMP\"" EXIT
+  if [ -z "$MAFIA_VERSION" ]; then
```
Comment (nhibberd) [inline script/mafia]: `! -z`

**Comment by nhibberd** [comment]
:rocket: after :bangbang:

**Pull request merged**